apps/launcher/main.cpp

```diff
@@ -200,7 +200,14 @@ int main(int, char** argv)
     }
 
     if (runFreeablo)
-        system((boost::filesystem::system_complete(argv[0]).parent_path() / "freeablo").string().c_str());
+    {
+#ifdef _WIN32
+        std::string path = "\"" + (boost::filesystem::system_complete(argv[0]).parent_path() / "freeablo").string() + "\"";
```

Comment:
Is this for spaces in directory path?
If so:
    It might have same issue on 64 bit windows and posix..
    `boost::filesystem` probably has something portable for escaping spaces

Reply:
The closest thing I could find would be to use a regex to escape the spaces. Is this ok?

Reply:
And I don't believe POSIX has this issue. I'll check this later.

Reply:
That's odd, posix shell should require spaces escaped too..
Regex is a bit overkill.
Maybe just escape path for any OS. otherwise nevermind

Reply:
Actually, you're right. Done.

Reply:
Did you test this on windows? Won't the \\ show up as a directory separator there?

Reply:
Ugh. I'm gonna open another pr for that.